src/PKSim.Presentation/Services/BuildingBlockIconRetriever.cs

```diff
@@ -71,10 +71,10 @@ public ApplicationIcon IconFor(ISimulationComparison simulationComparison)
 
       public ApplicationIcon IconFor(QualificationPlan qualificationPlan) => ApplicationIcons.Formula;
 
-      private string speciesNameFrom(UsedBuildingBlock usedBuildingBlock)
+      private string speciesIconFrom(UsedBuildingBlock usedBuildingBlock)
       {
          var individual = _buildingBlockRepository.ById(usedBuildingBlock.TemplateId) as Individual;
-         return individual == null ? usedBuildingBlock.BuildingBlockType.ToString() : individual.Species.Name;
+         return individual == null ? usedBuildingBlock.BuildingBlockType.ToString() : individual.Species.Icon;
```

Comment:
why not just `individual.Icon` ?

Reply:
fair enough. I did not want to change too much. This should be exactly identical